task7.js

```diff
@@ -0,0 +1,25 @@
+// Write a function checkNumber(n) that returns 'Positive', 'Negative', or 'Zero' based on the input number.
+// Sample output:
+// checkNumber(5) => Positive
+// checkNumber(0) => Zero
+
+const checkNumber = (n) => {
+  if (typeof n !== "number" || Number.isNaN(n)) {
+    throw new Error("Provided value must be a valid integer.");
+  }
+
+  return n > 0 ? "Positive" : n < 0 ? "Negative" : "Zero";
```

Comment:
try to avoid nester ternary conditions
for that case u can resolve it by `creating separate if for zero` or` create separate value for n < 0 condition`